Source/WebCore/testing/Internals.cpp

```diff
@@ -2985,6 +2985,8 @@ RefPtr<WindowProxy> Internals::openDummyInspectorFrontend(const String& url)
     auto* inspectedPage = contextDocument()->frame()->page();
     auto* localMainFrame = dynamicDowncast<LocalFrame>(inspectedPage->mainFrame());
     auto* window = localMainFrame ? localMainFrame->document()->domWindow() : nullptr;
```

Comment:
Seems like we should early exit if `localMainFrame` is null rather than using the `?:` operator.